frontend/src/components/UserListComponent.vue

```diff
@@ -43,6 +51,39 @@ export default {
     const csvc = commonService()
     const isFollow = ref(false)
     const { link } = config()
+    const userId = JSON.parse(csvc.getUserAndToken('user')).user_id
+
+    const followed = computed(() => {
+      const isFollow = props.user.following.filter(q => q.followed_id === userId)
+      if (isFollow.length) {
+        return 'UF'
+      }
+      const followBack = props.user.followed.filter(q => q.following_id === userId)
+      if (followBack.length) {
+        return 'FB'
+      }
+      return 'FF'
```

Comment:
Same comments from other PR, create constants for these values

Reply:
okay miss